pages/iOS-walkthrough.md

```diff
@@ -41,10 +41,12 @@ pod install
 
 You can know more about setting up a Pod file from [the CocoaPods documentation](https://guides.cocoapods.org/using/the-podfile.html).
 
-If you are not using CocoaPods, you can download the latest version of the framework from [our releases](https://github.com/tangrams/tangram-es/releases), then simply drag and drop the Framework into your iOS project. Then, from your poject settings under `Embedded Binaries`, click on `+` to add `TangramMap.framework`, and make sure that the framework appears in the build phases with _Code Sign On Copy_ is checked.
+If you are not using CocoaPods, you can download the latest version of the framework from [our releases](https://github.com/tangrams/tangram-es/releases), then simply drag and drop the Framework into your iOS project. Then, from your poject settings under `Embedded Binaries`, click on `+` to add `TangramMap.framework`, and make sure that the framework appears in the build phases with _Code Sign On Copy_ checked.
 
 ![](images/XCode-Codesign.png)
 
+>If you are using CocoaPods, make sure to open your `xcworkspace` project (not `xcodeproj`).
```

Comment:
If this is documented somewhere in cocoapod documentation, we should reference that here.